src/test/ui/span/issue-39018.stderr

```diff
@@ -59,11 +62,11 @@ LL |     let _ = &a + &b;
    |             |  |
    |             |  `+` cannot be used to concatenate two `&str` strings
    |             &String
+   |             help: remove the borrow to obtain an owned `String`
```

Comment:
Should I make this a `span_suggestion_verbose`?

Reply:
It might be a bit clearer as a verbose suggestion.